### ISSUE TITLE
[Request] Increase visual contrast in blog post frame

### ISSUE THREAD

**Issue by Sloofy** [int-72h/oftoast-godot#14]
### Summary
In places such as the blog post, the dark purple frame can be hard to make out and blend too easily into the background. At a quick glance, it can seem as though the placeholder thumbnails are floating images without clearly defined tabs separating each one individually. Even with fairly good eyesight, this can prove to be a problem. The frame itself should be updated to use a darker purple or the background should be adjusted to use a lighter colour instead.

**Current behaviour:**
![Screenshot_20230128_092526](https://user-images.githubusercontent.com/18003771/215280862-47f6ea91-3419-441c-a973-9d1108b5cb12.png)

**Proposed behaviour (lighter background, same frame colour):**
![proposed1](https://user-images.githubusercontent.com/18003771/215282166-d272533a-9a93-4e60-9be2-236c2aad4da8.png)

**Proposed behaviour (same background, darker frame colour):**
![proposed2](https://user-images.githubusercontent.com/18003771/215282303-3f197d76-20b2-4036-b71d-fe6e7afbc311.png)

**Comment by int-72h** [comment]
OFToast's design is based on Pont's mockups of it which it follows pretty closely.

**Comment by Sloofy** [comment]
> OFToast's design is based on Pont's mockups of it which it follows pretty closely.

Fair enough. The UI currently follows very closely to the themeing of the game's aesthetic as a whole, which I appreciate. Though, I think there's still an underlying issue of accessibility while using the software that should be taken into consideration. The proposed behaviour examples go to an extreme with how much should be adjusted to illustrate a point. To that end, I don't think we'd ever see pink as the new de facto OF colour, but it does highlight how hard it can be to read the UI at times.

**Comment by Sloofy** [comment]
Looking at this now, the advanced options menu uses a much better colour for its frame (#3d2951) which could likely replace the current blog post frame colour (#432d59). The blog is also the only location in toast to have this colour - everywhere else seems to use #3d2951. So, in a way, this could be argued as a parity fix.
